### PR TITLE
Update windows-autopilot-reset-remote.md, issue 3129

### PR DESCRIPTION
**Pull request by MaratMussabekov**
issue #3129 
added clarification related to hybrid deployments

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/deployment/windows-autopilot/windows-autopilot-reset.md [hunk 1]
```diff
@@ -33,6 +33,9 @@ Windows Autopilot Reset will block the user from accessing the desktop until thi
 >[!IMPORTANT] 
 >To reestablish Wi-Fi connectivity after reset, make sure the **Connect automatically** box is checked for the device's wireless network connection. 
 
+>[!NOTE]
+>The Autopilot Reset does not support Hybrid Azure AD joined devices.
```
Comment (JohanFreelancer9) [inline windows/deployment/windows-autopilot/windows-autopilot-reset.md]: Copy edit - no changes.

**Comment by mypil** [comment]
@greg-lindsay - can you please assist to check, approve and merge the content for this PR based on issue #3129?

Thank you.

**Comment by mypil** [comment]
@greg-lindsay - I would like to follow up on this. Can you please assist to check, approve and merge the content for this PR based on issue #3129?

Thank you.

**Comment by mypil** [comment]
@greg-lindsay - I would like to follow up on this. Can you please assist to check, approve and merge the content for this PR based on issue #3129?

Thank you.

**Pull request merged**